README.md

```diff
@@ -13,6 +13,18 @@ _Please note that although the below examples are written in interactive style,
 actually work in the interactive interpreter._
 
 
+####API changes in 1.2
+From version 1.2, the behavior of parallel filter method was changed to same with CPython's built-in filter.
+The differences with previous version are as follows:
```

Comment:
Did you think of perhaps just writing in the present tense? Like in "the behavior is change to" Also perhaps we can just say "the built-in `filer`".